activate AWS 11.3.3 and deprecate 11.3.2 (#309)
giantswarm/releases: release-notes/aws/v11.3.3.md

## :zap: Giant Swarm Release v11.3.3 for AWS :zap:

**If you are upgrading from 11.3.2, upgrading to this release will not roll your nodes. It will only update the apps.**

> This release fixes an issue where `cert-manager` could be killed for exceeding its memory limit.

Reviewer: @glitchcrab thanks for drafting! could you add some implication to customer, how does it negatively affect them when cert manager is killed?

Author: done - english is hard :see_no_evil:

## :zap: Giant Swarm Release v11.3.3 for AWS :zap:

**If you are upgrading from 11.3.2, upgrading to this release will not roll your nodes. It will only update the apps.**

> This release fixes an issue where `cert-manager` could be killed for exceeding its memory limit. This in turn means that SSL certificates

Reviewer: ```suggestion
This release fixes an issue where `cert-manager` could be killed for exceeding its memory limit. This stops SSL certificates from being automatically renewed, causing expired certificates marking your site as not secure.
```

@glitchcrab is my suggestion too obvious? hahaha

wouldn't be automatically renewed and could therefore expire.

> 

Reviewer: ```suggestion
Note for SEs: This release contains an external-dns fix introduced in [11.3.1](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.3.2.md#zap-giant-swarm-release-v1132-for-aws-zap). It requires manual intervention for cluster upgrades in China to work. When upgrading a cluster, existing ingress A+TXT record sets do not get replaced with CNAME+TXT record sets even when external-dns is configured with CNAMEs as preferred. After upgrading, delete the ingress A+TXT record sets. external-dns will then automatically create CNAME+TXT record sets.
```

@sslavic @glitchcrab is this required here?

Author: hmm that's a good point - it might be a good idea to retain that

## :zap: Giant Swarm Release v11.3.3 for AWS :zap:

**If you are upgrading from 11.3.2, upgrading to this release will not roll your nodes. It will only update the apps.**

This release fixes an issue where `cert-manager` could be killed for exceeding its memory limit. This stops SSL certificates from being
automatically renewed, which would cause expired certificates to show your site as not secure.

> Note for SEs: This release contains an external-dns fix introduced in [11.3.1](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.3.2.md#zap-giant-swarm-release-v1132-for-aws-zap). It requires manual intervention for cluster upgrades in China to work. When upgrading a cluster, existing ingress A+TXT record sets do not get replaced with CNAME+TXT record sets even when external-dns is configured with CNAMEs as preferred. After upgrading, delete the ingress A+TXT record sets. external-dns will then automatically create CNAME+TXT record sets.

Reviewer: ```suggestion
**Note for SEs:** This release contains an external-dns fix introduced in [11.3.1](https://github.com/giantswarm/releases/blob/master/release-notes/aws/v11.3.2.md#zap-giant-swarm-release-v1132-for-aws-zap). It requires manual intervention for cluster upgrades in China to work. When upgrading a cluster, existing ingress A+TXT record sets do not get replaced with CNAME+TXT record sets even when external-dns is configured with CNAMEs as preferred. After upgrading, delete the ingress A+TXT record sets. external-dns will then automatically create CNAME+TXT record sets.
```
